.github/workflows/reviewdog.yaml

```
@@ -67,7 +67,7 @@ jobs:
         with:
           token: ${{ secrets.GH_ACCESS_TOKEN }}
```

We might also try changing the token here to 

```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```